### PR TITLE
Added new keyboard model [Skyloong GK61 pro_ds] with separate spacebar

### PR DESCRIPTION
**Pull request by Norp1x**
## Description

Added new keyboard model [Skyloong GK61 pro_ds] with separate spacebar, which is hard to find on internet

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by italovieira** [comment]
This is a duplicate of #22988.

**Comment by Norp1x** [comment]
> This is a duplicate of #22988.

Not at all... My PR was first. The other one is a duplicate of this one ;)

**Comment by italovieira** [comment]
> > This is a duplicate of #22988.
> 
> Not at all... My PR was first. The other one is a duplicate of this one ;) 

Are you sure? Look at the ids or creation dates.

**Comment by Norp1x** [comment]
> > > This is a duplicate of #22988.
> > 
> > 
> > Not at all... My PR was first. The other one is a duplicate of this one ;)
> 
> Are you sure? Look at the ids or creation dates.

Oh my! You're right ! I have no idea how do I missed that... I was looking for it everywhere. My bad :/

File: keyboards/skyloong/gk61/pro_ds/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+     case MO(3):
+      if (record->event.pressed) {
+       FN_MAC = 1;
+      } else {
+       FN_MAC = 0;
+      }
+      return true; // continue all further processing of this key
+
+     case TD(_DN_MU):
+         action = &tap_dance_actions[TD_INDEX(keycode)];
```
Comment (iamdanielv) [inline keyboards/skyloong/gk61/pro_ds/keymaps/via/keymap.c:229]: Should TD_INDEX be QK_TAP_DANCE_GET_INDEX?
```suggestion
         action = &tap_dance_actions[QK_TAP_DANCE_GET_INDEX(keycode)];
```

**Pull request closed without merging**